src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -10,20 +10,49 @@
     [org.broadinstitute.firecloud-ui.common.components :as comps]))
 
 
+
+(defn create-import-mc-url-path [workspaceNamespace workspaceName]
```

Comment:
Since this function is in the `method-config-importer` namespace, it seems redundant to include "import-mc" in the function's name.